### PR TITLE
Add Ubuntu Jammy

### PR DESCRIPTION
**Pull request by artembo**
Created the new Dockerfile to build Ubuntu Jammy based on Ubuntu Focal.

Squashed docker layers into one RUN step to reduce images size.

Part of tarantool/tarantool-qa#237

### CODE REVIEW

File: ubuntu/jammy/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
+ARG ARCH=
+FROM ${ARCH}ubuntu:impish
```
Comment (Totktonada) [inline ubuntu/jammy/Dockerfile]: ??

Comment (artembo) [reply ubuntu/jammy/Dockerfile]: It was a draft PR for @ylobankov to show the new layer structure. Fixed.

File: ubuntu/jammy/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    gdb \
+    ccache \
+    devscripts \
+    debhelper \
+    cdbs \
+    fakeroot \
+    lintian \
+    equivs \
+    rpm \
+    alien  \
```
Comment (ylobankov) [inline ubuntu/jammy/Dockerfile]: Please remove one redundant space before `\`.

File: ubuntu/jammy/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    && apt-get clean  \
```
Comment (ylobankov) [inline ubuntu/jammy/Dockerfile]: ditto

**Review by ylobankov — APPROVED** [review]
LGTM

**Review by Totktonada — APPROVED** [review]
LGTM.

**Pull request merged**